R/utils-yaml.R

```diff
@@ -0,0 +1,3 @@
+read_yaml <- function(path) {
```

Comment:
Reading your latest blog post, I don't know which approach you want to follow for the roxygen of such internal function. I'm suggesting `@keywords internal` here, but I found the @devtag approach interesting.

```suggestion
#' Alias for yaml::read_yaml with a handler for 'seq'
#'
#' @param path Path to yaml file
#' @keywords internal
read_yaml <- function(path) {
```